cmd/scripts/k8s/setup-host.sh

```diff
@@ -6,28 +6,26 @@ cp /usr/local/bin/build-start-daemon.sh /host/build-start-daemon.sh
 
 chroot /host <<"EOT"
 
-# of course, there's differences in the names of some of these packages
-YUM_PACKAGES="wget libnet-devel \
-    libnl3-devel libcap-devel libseccomp-devel gpgme-devel btrfs-progs-devel buildah criu"
+if [[ $SKIPSETUP -eq 1 ]]; then
+    cd /
+    IS_K8S=1 ./build-start-daemon.sh --systemctl --no-build
+    exit 0
+fi
 
-APT_PACKAGES="wget libnl-3-dev libnet-dev \
-    libbsd-dev libcap-dev pkg-config \
-    libgpgme-dev libseccomp-dev libbtrfs-dev buildah"
+YUM_PACKAGES=(wget libnet-devel libnl3-devel libcap-devel libseccomp-devel gpgme-devel btrfs-progs-devel buildah criu)
+APT_PACKAGES=(wget libnl-3-dev libnet-dev libbsd-dev libcap-dev pkg-config libgpgme-dev libseccomp-dev libbtrfs-dev buildah)
 
 install_apt_packages() {
     apt-get update
 
-    apt-get install -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" golang-github-containers-image golang-github-containers-common
+    apt-get install -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" golang-github-containers-image golang-github-containers-common
 
-    for pkg in $APT_PACKAGES; do
-        apt-get install -y $pkg || echo "Failed to install $pkg"
-    done
+    # install all packages at once
+    apt-get install -y "${APT_PACKAGES[@]}" || echo "Failed to install $pkg"
```

Comment:
my only beef with this is that previously, by iterating through packages, if a package didn't exist we would continue and install all the rest (instead of failing). Now that packages are trimmed in size though maybe it's not such a bad thing bc these are all necessary.